src/less/panel-dialog/base/panel-dialog.less

```diff
@@ -37,21 +38,16 @@
     .dialog-footer-content();
 }
 
-.panel-dialog__back,
-.panel-dialog__close {
-    .customColorProperty(dialog-back-close-foreground-color);
-
+// inherits from .icon-btn
+button.panel-dialog__close {
```

Comment:
Should we also have a secondary button class here?

Reply:
You mean for back button?

Reply:
For like a reset button or a secondary button on the header. Are we supporting that still?

Reply:
We are using `fake-link` now in 11.0.0

Reply:
I didn't really make any changes here in that regard... only removing selector for back button where it didn't make sense (in lightbox and drawer)

Reply:
Looking through the stories, looks like I broke something with `height: auto`. Fixing..

Reply:
![Screen Shot 2020-09-10 at 6 02 31 PM](https://user-images.githubusercontent.com/38065/92835219-d2290d00-f38f-11ea-8b03-0fb7f3aa970d.png)

Reply:
Fixed the glaring alignment issues, but it's still not perfect. We are going to follow this up in a separate PR.